fibbingnode/southbound/fibbing.py

```diff
@@ -323,6 +323,14 @@ def proxy_remove(self, points):
     def proxy_connected(self, session):
         self.root.send_lsdblog_to(session)
 
+        if self.routes:
```

Comment:
This change would be super useful!
Could you try to:
* Limit the route removal to only the ones that were advertized by the application.
* Perform that removal when an application disconnects